### PR TITLE
try harder to ensure that a held object is active in bullet

### PR DESCRIPTION
**Pull request by sethalves**
- try harder to ensure that a held object is active in bullet

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: b873d92469edff1e22670da7067d9cf430079112 available at: https://gist.github.com/0f5b07ca2f68faedee56<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6783.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6783.dmg

**Comment by ZappoMan** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: b873d92469edff1e22670da7067d9cf430079112 available at: https://gist.github.com/0f5b07ca2f68faedee56<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6783.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6783.dmg

File: libraries/physics/src/ObjectAction.cpp [hunk 1]
```diff
@@ -237,10 +237,15 @@ void ObjectAction::setAngularVelocity(glm::vec3 angularVelocity) {
...
     auto rigidBody = getRigidBody();
     if (rigidBody) {
-        rigidBody->activate();
+        rigidBody->activate(forceActivation);
+        if (! rigidBody->isActive()) {
+            qDebug() << "ObjectAction::activateBody -- rigidBody->activate() failed" << (void*)rigidBody;
+        }
+    } else {
+        qDebug() << "ObjectAction::activateBody -- no rigid body" << (void*)rigidBody;
     }
```
Comment (AndrewMeadows) [inline libraries/physics/src/ObjectAction.cpp]: Do we really need the logging?

Comment (sethalves) [inline libraries/physics/src/ObjectAction.cpp]: It's not strictly required, but if either of those ifs fail, I'd like to know.  Would asserts be better?

**Comment by hifi-gustavo** [comment]
Build console output for commit: cf9af462b5958aa8a448ded214af836f6c174ddd available at: https://gist.github.com/0f5b07ca2f68faedee56<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6783.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6783.dmg

**Pull request merged**